doc/scheduler.md

```diff
@@ -12,6 +12,16 @@ This is the interface you need to implement.
 
 ~~~ ruby
 class Scheduler
+  # Wait for the specified process ID to exit.
+  # This hook is optional.
+  # @parameter pid [Integer] The process ID to wait for.
+  # @parameter flags [Integer] A bitmask of flags suitable for `Process::Status.wait`.
```

Comment:
It should mention it needs to return a Process::Status object using `Process::Status.wait(pid, flags)`.

Reply:
Ah yeah good idea.